frontend/src/homePage/CodeGroups.jsx

```diff
@@ -0,0 +1,124 @@
+import { useState, useEffect } from "react";
```

Comment:
Since you'll have more groups-related code in the future, I would create a separate folder for the code groups files

Reply:
This code group component is a sub-component for the home page. I wanted to have a group workstation folder for every components in the UI of the page the user will see when a group is clicked.

Reply:
Ok, I think I would rename this file then, since right now by the name I would assume this is part of the workstation. Something like 'createCodeGroup' would be clearer naming